notebooks/en/analyzing_art_with_hf_and_fiftyone.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Small typo here (italicized so you can see it easier) :)</p><p> </p><p>"This will allow us to see how the images cluster based on their <em>style</em>, genre, and artist."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Thanks! Fixed.